### ISSUE TITLE
Finish Proposal Report (team: Gretel @gtmx23  and Sophia @zth96)

### ISSUE THREAD

**Issue by gtmx23** [coruubc/diverse_price_prediction#13]
Details
- [Jupyter book](https://pages.github.ubc.ca/mds-2023-24/DSCI_591_capstone-proj_students/deliverables/1-proposal-presentation-and-report.html)

Distribution
-  @gtmx23 to do Executive Summary and Intro section
   - Executive Summary section is expected to change after Wed
- @zth96 to do DS Techniques and Milestone section
- Someone to do final review 

Format
- To work on draft in Google docs https://docs.google.com/document/d/1ikKQfeHOduTa2DwYA8MHhO0VNIYPxGonibGvuzfvBeY/edit?usp=sharing
-  To submit draft in HTML/PDF by **7 May noon** to Alexi on internal channel
- To convert to Rmd to be uploaded in `reports` section of repository
- Final submission on **10 May (Fri) 6 pm**:
  - Link to repo Rmd to be submitted on Canvas
  - Also to be shared on external Slack channel

**Comment by gtmx23** [comment]
Draft deadlines:
- 9 am 7 May
- @gtmx23 can review for overall language, @zth96 shall we look over each other's sections too? I would feel better knowing a second pair of eyes has checked through my work haha. @sho-i98 or @fohy24 feel free to review for content too!

**Comment by zth96** [comment]
Thanks Gretel @gtmx23 , it would definitely be nice for us to cross-check and read through each other's part. I have just posted my draft on the google doc link. https://docs.google.com/document/d/1ikKQfeHOduTa2DwYA8MHhO0VNIYPxGonibGvuzfvBeY/edit

This is the draft version of the 3.3 and 3.4. I have also included a conclusion. 
I'm thinking about maybe including some graphics for dataset overview and still working on having that added. 
Regarding word count, it's quite long as of now 1500 words. 
I'll keep working on editing them and definitely need to cut off some sections. 
I do not think my section would have anything to be quoted except graphics.

Feel free to review and comment.
